scripts/insert_into_appointments.sql

```diff
@@ -1 +1,7 @@
 -- insert appointments creating connections between doctors and patients
+insert into appointments(patient_id, doctor_id, date, time, comment)
+  values (8, 4,'August 12, 2016', '10am', ''),
+          (5, 6, 'August 17, 2016', '1pm', ''),
+          (8, 6, 'August 12, 2016', '10am', ''),
+          (7, 4, 'August 17, 2016', '1pm', '')
+          ;
```

Comment:
This setup is generally right. Hard-coding values like this will work, so I won't take points off, but it probably would have been a better idea to look up the specific patients and doctors from their respective tables.